lib/web_of_science/harvester.rb

```diff
@@ -15,6 +15,7 @@ def harvest(authors)
       authors.each do |author|
         process_author(author)
         author_success += 1
+        GC.start # suggest that now is a good time to clean up some memory
```

Comment:
I'm skeptical of adding explicit `GC` calls without concrete measurement that it helps.  As frequently as not, it can hurt.  `GC.start` is a process-blocking FULL SWEEP, not a "suggestion".  It is **guaranteed** to add latency every time, so the questions is "to what benefit?"

During debugging, you can interrogate the state w/ `GC.stat` to examine what the allocation stats are before/after running to see if you are really saving a significant amount.  More powerfully, you can try [`GC::Profiler`](http://ruby-doc.org/core-2.2.0/GC/Profiler.html).

I don't see anything we're doing here that makes us smarter than Ruby itself about the timing of collection.  Processing some authors will use a lot of memory and others comparably little.  So it seems weird to force Ruby's hand here.  AFAIK, our problem was that our process was **actually using** too much memory.  We have not determined that it was failing to recover assigned memory.  Please let me know if I'm missing something.

Reply:
Understood, expected this to be a point of discussion.  I've read a little about `GC`, but I have never used it and do not understand it.  I added it in a couple places because it's also used in the SW harvester and didn't want to miss an opportunity to clean up memory (if possible).

Reply:
concur with Joe - I think we should hold off on doing an explicit GC for now unless we have evidence that it is helping (even though the current harvest is running fine with this code, the other changes are more likely responsible for this)

Reply:
Also of note, I have tried using the GC explicitly in the past with pre-assembly and after some experimentation so no difference in memory usage.